### PR TITLE
Split formatters to simple functions

### PR DESCRIPTION
**Pull request by iliabv**
After addition of `combineFormatters` I couldn't stop myself from creating this PR.
Formatters now just a simple functions. It's very easy to create a vast amount of complex formatters by combinations of simple formatters with different parameters and in different positions.
Code and tests now simpler, but from user's point of view it (maybe) harder to understand.

### CODE REVIEW

File: src/formatters/date.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+ */
+export default function create(createDate = createDatePart) {
+    /**
+     * @param {String} method
+     * @param {String} [prefix]
+     * @param {Array<*>} args
+     * @returns {Array<String>}
+     */
+    return function formatDate(method, prefix, args) {
+        return [createDate()].concat(args);
```
Comment (acvetkov) [inline src/formatters/date.js]: You can write it in es6 way

``` js
return [createDate(), ...args];
```

File: README.md [hunk 2]
```diff
@@ -32,11 +51,11 @@ otherLogger.info('Hi, %o', {world: 'Earth', age: 9999999999})
 // this methods are applied to both logger and otherLogger
```
Comment (acvetkov) [inline README.md:51]: Logger is `undefined` in example
https://github.com/ilya-babanov/lggr/pull/3/files#diff-04c6e90faac2675aa89e2176d2eec7d8R41

File: src/writers/console.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
+/**
+ * Writes formatted arguments to console;
+ */
+
+export default function create() {
+    var apply = Function.prototype.apply;
+    var _console = console ? console : {};
```
Comment (acvetkov) [inline src/writers/console.js]: `const`

File: README.md (hunk 2 above)
Comment (iliabv) [inline README.md:51]: Fixed

**Comment by iliabv** [comment]
@vitalets ping

**Comment by vitalets** [comment]
:+1: 
I think it would be also usefull to describe each formatter in readme with sample output

**Comment by acvetkov** [comment]
I think it would be great to create detailed description for lggr settings and usage examples.
https://github.com/ilya-babanov/lggr/issues/4

**Pull request merged**